component_sdk/python/kfp_component/_base_op.py

```
@@ -0,0 +1,213 @@
+            return
+
+        self._argo_workflow_name = pod.metadata.labels.get('workflows.argoproj.io/workflow', None)
+        argo_node_name = pod.metadata.annotations.get('workflows.argoproj.io/node-name', None)
+        if argo_node_name:
+            self._argo_node_name = re.sub(r'\s+\(\d\)', '', argo_node_name)
+
+    def _load_staging_location(self):
+        tmp_location = os.environ.get('KFP_TMP_LOCATION', None)
+        if not tmp_location:
+            return
+
+        if not self._argo_workflow_name or not self._argo_node_name:
+            return
+
+        gs_prefix = 'gs://'
+        if tmp_location.startswith(gs_prefix):
+            tmp_location = tmp_location[len(gs_prefix):]
+        splits = tmp_location.split('/', 1)
+        self._staging_bucket = splits[0]
+        blob_path_prefix = ''
+        if len(splits) == 2:
+            blob_path_prefix = splits[1]
+        self._staging_path = os.path.join(blob_path_prefix, 'tmp/{}/{}/{}'.format(
+            self._argo_workflow_name,
+            self._argo_node_name,
+            self.name
+        ))
+
+    def _load_staging_states(self):
+        if not self._staging_bucket or not self._staging_path:
+            return
+
+        try:
+            bucket = self._storage.get_bucket(self._staging_bucket)
+        except google.cloud.exceptions.NotFound:
+            logging.error('Unable to find staging bucket {}.'.format(self._staging_bucket))
+            return
+
+        blob = bucket.blob(self._staging_path)
+
+        try:
+            states_json = blob.download_as_string()
+        except google.cloud.exceptions.NotFound:
+            logging.info('Unable to find staging blob {}.'.format(self._staging_path))
+            return
+
+        try:
+            self.staging_states = json.loads(states_json)
+        except ValueError as e:
+            logging.error('Unable to decode staging states: {}. Error: {}.'.format(states_json, e))
+            return
```

Is it OK to ignore this error?

---

This code is removed.